app/populate/data/graduation.js

```diff
@@ -0,0 +1,49 @@
+module.exports = function () {
+  return  [{      
+      "curso": "Bacharelado em Ciência e Tecnologia",
+      "grade": "2017",
+      "credits_total": 190,
+      "free_credits_number": 43,
+      "limited_credits_number": 57,
+      "locked": true,
```

Comment:
Codacy has a fix for the issue: [Expected indentation of 4 spaces but found 6.](https://app.codacy.com/gh/ufabc-next/ufabc-next-server/pullRequest?prid=8555380)
```suggestion
    "locked": true,
```